api/v1alpha1/envoyproxy_metric_types.go

```diff
@@ -41,6 +41,19 @@ type ProxyMetrics struct {
 	//
 	// +optional
 	EnableRequestResponseSizesStats *bool `json:"enableRequestResponseSizesStats,omitempty"`
+
+	// ClusterStatName defines the value of cluster alt_stat_name, determining how cluster stats are named.
+	// The supported operators for this pattern are:
+	// %ROUTE%: name of Gateway API xRoute resource in <NAMESPACE>/<NAME> format
+	// %ROUTE_KIND%: kind of Gateway API xRoute resource
+	// %ROUTE_RULE%: name of the Gateway API xRoute section
+	// %ROUTE_RULE_NUMBER%: name of the Gateway API xRoute section
+	// %BACKEND_REFS%: names of all backends referenced in <NAMESPACE>/<NAME>,<NAMESPACE>/<NAME>,... format
```

Comment:
can you check if `,` will be accepted by prom ?
we may need to use another separator here

Reply:
In general, prometheus docs (https://prometheus.io/docs/concepts/data_model/) say:

> Label values MAY contain any UTF-8 characters.

However, to be on the safe side, we can use a delimiter like `|` which is used for cluster names by other (e.g. istio), to reduce compatibility risks with other observability systems.

Reply:
sgtm